src/core/services/eth-service.ts

```diff
@@ -31,6 +31,11 @@ export default class EthService {
     return block && new EthqlBlock(block);
   }
 
+  public async fetchOmmerBlock(id: string, index: number): Promise<EthqlOmmerBlock> {
```

Comment:
A clearer name for the `id` arg would be `blockHashOrNumber`, and it can be a string (block hash) or a number (block number).